comixed-webui/src/assets/i18n/es/comic-metadata.json

```diff
@@ -97,11 +98,11 @@
       "no-match": "No"
     },
     "tooltip": {
-      "cancel-scraping": "Deja de raspar cómics.",
-      "reject-scraping-issue": "Rechazar este problema.",
-      "select-scraping-issue": "Seleccione este problema.",
-      "show-comic-details": "Mostrar detalles",
-      "skip-cache": "Raspado será {enabled, select, true{skip} other{use}} la caché de raspado.",
+      "cancel-scraping": "Stop scraping comics.",
+      "reject-scraping-issue": "Reject this issue.",
+      "select-scraping-issue": "Select this issue.",
+      "show-comic-details": "Show Details",
+      "skip-cache": "Scraping will {enabled, select, true{skip} other{use}} the scraping cache.",
```

Comment:
Why change these elements ?
A remark in passing, I am not sure that the Spanish translation is relevant. "Issue" in English corresponds more to "número" than to "problema" which would be closer to "problem".